dali/pipeline/executor/workspace_policy.h

```diff
@@ -390,14 +408,15 @@ struct AOT_WS_Policy<SeparateQueuePolicy> {
   template <OpType op_type, OpType group_as = op_type, typename T>
   void PlaceWorkspace(T &workspaces, QueueIdxs idxs, const OpGraph &graph,
                       const std::vector<tensor_data_store_queue_t> &tensor_to_store_queue,
-                      cudaStream_t mixed_op_stream, cudaStream_t gpu_op_stream,
+                      ThreadPool *thread_pool, cudaStream_t mixed_op_stream,
```

Comment:
Move each argument to its own line or group them in a meaningful way.

Reply:
done